### ISSUE TITLE
MySQL 8 on Windows

### ISSUE THREAD

**Issue by jackpoz** [appveyor/ci#3834]
It would be nice to include MySQL 8 in some windows machine. According to https://www.appveyor.com/docs/services-databases/#mysql the only MySQL version included on windows in 5.7 .

**Comment by Shauren** [comment]
This becomes more relevant now that 5.7 reaches its EOL date

**Comment by jackpoz** [comment]
@OwenMcDonnell it feels like this issue got lost. MySQL 5.7 has EOL set on October 21, 2023 , after which everyone will have to move to MySQL 8.

**Comment by OwenMcDonnell** [comment]
You're right, it did get lost! Apologies, but I'll be sure to include it in the next image update.

**Comment by dveeden** [comment]
The doc still only lists MySQL 5.7, which by now has reached EOL.

https://github.com/appveyor/website/blob/ea7b711429e1f509731c8b90bd136b73240d5013/src/docs/windows-images-software.md?plain=1#L1278 

Note that MySQL 8.1.0 and 8.2.0 were released recently as well. While 8.0.x is stable the 8.1.x and 8.2.x releases are innovation releases. 

https://github.com/appveyor/ci/issues/3894 suggests that MySQL 8.0 might be included already for some images.

**Comment by OwenMcDonnell** [comment]
Docs will be updated soon to reflect that MySQL 8.0 is indeed now pre-installed on `Visual Studio 2019` and `Visual Studio 2022` images. Unfortunately, this update doesn't yet work with the `services:` config syntax. Simple workaround for now is to add the following to your config. 
```
init:
  - ps: Start-Service MySQL80
 ```

We'll consider newer releases at next image update.

**Comment by Bartleby2718** [comment]
I realized that the same problem exists for Postgres 15. If you want to start both MySQL 8.0 and Postgres 15 on a Windows image, you can do
```yaml
init:
  - ps: Start-Service MySQL80, postgresql-x64-15
```
because `Start-Service` accepts a comma-separated list of service names.
